blueprints/resource/index.js

```diff
@@ -15,9 +14,11 @@ module.exports = {
   uninstall: function(options) {
     return this._process('uninstall', options);
   },
-  
+
   _processBlueprint: function(type, name, options) {
     var mainBlueprint = this.lookupBlueprint(name);
+
+    var that = this;
```

Comment:
yeah, but then the existing mentions of `this` in the code would be differently bound and I wasn't sure if that would work as well